src/core/stream.js

```diff
@@ -739,15 +739,16 @@ var PredictorStream = (function PredictorStreamClosure() {
     var pos = bufferLength;
     var i;
 
-    if (bits === 1) {
+    /* Optimized version of the loop in the "else"-branch
+    for 1 bit-per-component and 1 color TIFF images */
```

Comment:
Nit: Please note that we only use this kind of comment style for function/method documentation (and copyright notices). For other comments, the correct form would be this (and please also remember to add a period at the end of the sentence):
```javascript
// Optimized version of the loop in the "else"-branch
// for 1 bit-per-component and 1 color TIFF images.
```